README.md

```diff
@@ -1,122 +1,132 @@
+in the env file are set as below
+
+There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
+
+| Variable                | Description                                                                                                                             |
+| ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| SECRET_KEY              | [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie. |
+| MONGO_CONNECTION_STRING | The access string for editing the azure mongo db                                                                                        |
+| MONGO_DB_NAME           | The name of the database used to store the app data                                                                                     |
+| OAUTH_APP_ID            | The ID of the github OAuth app                                                                                                          |
+| OAUTH_APP_SECRET        | The secret key for the github OAuth app                                                                                                 |
+
+## Running the App
+
+Once the all dependencies have been installed, start the Flask app in development mode within the Poetry environment by running:
+
+```bash
+$ poetry run flask run
+```
+
+You should see output similar to the following:
+
+```bash
+ * Serving Flask app "app" (lazy loading)
+ * Environment: development
+ * Debug mode: on
+ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
+ * Restarting with fsevents reloader
+ * Debugger is active!
+ * Debugger PIN: 226-556-590
+```
+
+Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+### Running the app using ansible
+
+To run the app using ansible copy the `/ansible/controller_files` folder to the controller node
+
+Then update the inventory to include any additional host nodes and run the below command from inside the `controller_files` folder
+
+`ansible-playbook playbook.yml -i inventory`
+
+### Running the app using docker
+
+To run the app using docker ensure docker daemon is running on your machine then
+
+#### Development
+
+Build the container by running `docker-compose -f docker-compose-dev.yml build`
+
+Run the container using `docker-compose -f docker-compose-dev.yml -p 'todo-app-dev' up`
+
+The dev site will then be available on `localhost:5000`
+
+#### Production
+
+Build the container by running `docker-compose -f docker-compose-prod.yml build`
+
+Run the container using `docker-compose -f docker-compose-prod.yml -p 'todo-app-prod' up`

```

Comment:
This step fails for me with 
![image](https://user-images.githubusercontent.com/6737755/205093097-50f5aeeb-ad91-42a8-bb6b-8a8f708c62c0.png)
This happens running in bash and PowerShell
I get the same result with `docker run thomasjohnstonsoftwire/todo-app:latest`, which leads me to believe there is something up with the docker images themselves.

This currently prevents me from being able to run Kubernetes instructions later to reproduce a running Kubernetes environment. It needs to be remediated